react-client/src/Components/App.js

```diff
@@ -112,12 +107,19 @@ function App() {
   };
 
   const handleRemoveDialogBox = (idx) => {
-
     const newDomInfo = domInfo.filter((x, currentIdx) => currentIdx !== idx);
 
     setDomInfo(newDomInfo);
-  }
-  // const resetdom = () => setshowSwitch(true);
+  };
+
+  const handleSeemore = () => {
+    debugger
+   try {
```

Comment:
Why use try-catch? This is a bad practice to catch code errors and ignore them.

Reply:
noted already removed and modified